public/electron.js

```diff
@@ -98,7 +108,7 @@ function createWindow() {
   mainWindow.loadURL(
     isDev
       ? "http://localhost:3000"
```

Comment:
is there any reason to distinguish between dev and prod? If not, I would suggest to unify them. As choosing the port could be a feature handy in both environments.

Reply:
The difference here is that when launching electron on development we are not using a compiled electron instance. So we are not using the internal express server instance. Instead we are running it using react-hot-loaded server (yarn start) that is used for development